package.json

```diff
@@ -105,13 +108,15 @@
     "copy-webpack-plugin": "^4.2.0",
     "enzyme": "^3.6.0",
     "enzyme-adapter-react-16": "^1.5.0",
+    "expose-loader": "^0.7.3",
     "fast-glob": "^3.0.1",
     "html-loader": "^0.5.1",
     "is-plain-object": "^5.0.0",
     "jest-cli": "^23.6.0",
     "markdown-loader": "^5.1.0",
+    "sass-loader": "^6.0.5",
     "storybook-addon-jsx": "^5.4.0",
-    "webpack": "^2.6.1"
+    "webpack": "^3.12.0"
```

Comment:
This was necessary to get jquery to install at all. Thankfully the only change required to run this version of webpack was adding `expose-loader` and `sass-loader` as explicit dependencies (previously added via webpack-config) so it's not a big deal and won't affect anything.